src/main/java/org/betonquest/betonquest/modules/logger/custom/debug/config/DebugConfig.java

```diff
@@ -0,0 +1,82 @@
+package org.betonquest.betonquest.modules.logger.custom.debug.config;
+
+import java.io.IOException;
+
+/**
+ * This interface provides methods to manage settings related to debug settings.
+ */
+public interface DebugConfig {
```

Comment:
This interface violates the Interface Segregation Principle:
* `isDebugging`, `startDebug` and `stopDebug` are API for activity management (used as external API)
* `getExpireAfterMinutes` is implementation specific and the only true config part
* `addOnStartHandler`, `removeOnStartHandler`, `addOnStopHandler`, `removeOnStopHandler` and `PrePostRunnable` are API for lifecycle management (used by Handler implementations)